### ISSUE TITLE
Final Peer Review - ly389

### ISSUE THREAD

**Issue by Linna618** [pangolion/Mental-Project#15]
The project brought the topic that nowadays, people who suffers from mental health tend to avoid seeking for professional helps. To solve this problem, the project tries to find the reasons why peoples in tech industry tends to conceal their mental health conditions and avoid ask professionals to help them.
Pros:
1. The topic is very interesting. It’s related to our professional lives. The structure of the report is pretty good. The number of observations and features are enough. 
2. The project introduced comprehensively the number of the features, how they conducted encoding, how they deal with the missing values and how to avoid overfitting.
3. They tested their data with 6 models and briefly explained the models. It’s great that they used XGBoost model which is super powerful machine learning tools that improve the prediction accuracy. The model prediction analysis is comprehensive.
Further improvements:
1. There is no information about the data source, and the date of the data they used. They could briefly introduce what the features are about.
2. If they could explain the models with more math equations, that would be better and professional.
3. Data exploration and visualization could be added to help readers learn more about the data. They could also introduce the Weapon of Math Destruction and whether fairness is an important criterion to consider when choosing a model.
